dev-libs/libdbh/libdbh-5.0.22.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils
+
+MY_P=${PN}2-${PV}
+
+DESCRIPTION="a small library to create and manage 64-bit disk based hash tables"
+HOMEPAGE="https://www.gnu.org/software/libdbh/"
+SRC_URI="mirror://sourceforge/dbh/dbh/${PV}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	dev-util/gtk-doc-am
+	virtual/pkgconfig"
+
+S="${WORKDIR}"/${MY_P}
+
+pkg_setup() {
+	DOCS=( AUTHORS ChangeLog NEWS README TODO )
```

Comment:
There's no reason to define this in `pkg_setup` instead of global scope (or `local`-ly in `src_install`). Also, I think it matches what the default would do if you didn't declare it at all.